src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -500,10 +536,14 @@ private static void init() {}
          * Holds all fluids that are gaseous at room temperature.
          */
         public static final TagKey<Fluid> GASEOUS = tag("gaseous");
+        /**
+         * Holds all fluids related to honey.
+         */
+        public static final TagKey<Fluid> HONEY = tag("honey");
```

Comment:
Inclusion of honey here feels weird without also including soups, stews, and alike; all are fluids that forge does not implement but are common in mods. Potion is also a common fluid tag.

Reply:
Reason I added honey fluid tags was because Fabric actually has a honey fluid tag and that I know of 7 mods currently in 1.19.2+ that has honey fluids (Create, Inspiration, The Bumblezone, Cyclic, Tinkers, Productive Bees, Resourceful Bees)

Though I don't understand the soup and potion thing. Potion is its own item. And soup items can be holding any kind of fluid (like rabbit stew vs mushroom stew fluids)

Or is it that people are converting these items to fluids like a general `soup` fluid for all stews and a `potion` fluid for all potions and sticking that into a bucket? And how many mods are doing those things?

Reply:
Tinkers, Thermal, and a few other mods have a potion fluid, 250mb per bottle. Create also has a potion fluid but they use a different standard.

Mushroom soup, beetroot soup, and alike are all fluids in inspirations as well, and I've seen them in another mod before.

Reply:
To clarify, all potions are generally 1 fluid, using NBT. Everyone agreed on a NBT format.

Each soup is a separate fluid, because there are not that many soups (just 3) and its not backed by another registry.

Reply:
Forgot to ask, are bowls treated like bottles of 250mb per bowl?

Reply:
I'll go with bowls are 250mb for now. I added the fluid tags you asked for plus suspicious stew. lmk if the bowls are actually a different measurement and I'll fix the javadoc